packages/eslint-plugin-svelte/src/utils/element-types.ts

```diff
@@ -0,0 +1,140 @@
+export const voidElements = [
```

Comment:
Moved from removed `void-elemets.ts` file. It's actually different that the list that Svelte uses (https://github.com/sveltejs/svelte/blob/c32a91891fc647998f738371a012fcba97d56e50/packages/svelte/src/utils.js#L16-L33).